src/legacy/ui/public/vis/editors/default/agg_select.js

```diff
@@ -42,31 +42,50 @@ uiModules
       require: '^ngModel',
       template: function () {
         return `<vis-agg-select-react-wrapper
+            ng-if="setValidity"
```

Comment:
This prevents rendering `DefaultEditorAggSelect` component with undefined props during the first run.